boa/cli/mambabuild.py

```diff
@@ -78,7 +78,6 @@ def __init__(self, channels, platform):
         self.repos = []
```

Comment:
Line https://github.com/mamba-org/boa/pull/42/files#diff-6d845a6fe42e99697ca13ec691bcc8aa9b5e053dd3a837f53f3689209c734b6fR41

It complains with `F821 undefined name 'installed_pkg_recs'`, I don't know how to fix it :S